commit 376573aea0c1845738141071fc6db97a8f19637d
Author: Reto Gantenbein <reto.gantenbein@linuxmonk.ch>
Date:   Sat Jan 28 10:52:36 2023 +0100

    Revert "init: respect --prefix when installing systemd unit files"
    
    This reverts commit 9bfc897a0c6b010464dc490e955dbc1d1a66e4c3.

diff --git a/config/init/meson.build b/config/init/meson.build
index 8445955..5200570 100644
--- a/config/init/meson.build
+++ b/config/init/meson.build
@@ -2,7 +2,7 @@
 
 if 'systemd' in init_script
     systemd = dependency('systemd')
-    systemd_system_unit_dir = get_option('prefix') + systemd.get_pkgconfig_variable('systemdsystemunitdir')
+    systemd_system_unit_dir = systemd.get_pkgconfig_variable('systemdsystemunitdir')
     systemd_service = custom_target(
         'lxcfs.service',
         input: 'systemd/lxcfs.service.in',
